gt/main.c

```diff
@@ -81,6 +170,13 @@ gt_proc(void *arg)
 			continue;
 
 		for (i = 0; i < num_rx; i++) {
+			int ret;
+			uint8_t inner_ip_ver = 4;
```

Comment:
Why are you initializing `inner_ip_ver` since it'll receive a value later before its first use?